master/buildbot/test/fake/latent.py

```diff
@@ -165,19 +172,8 @@ class ControllableLatentWorker(AbstractLatentWorker):
 
     def __init__(self, name, controller, **kwargs):
         self._controller = controller
-        AbstractLatentWorker.__init__(self, name, None, **kwargs)
-
-    def checkConfig(self, name, _, **kwargs):
-        AbstractLatentWorker.checkConfig(
-            self, name, None,
-            build_wait_timeout=self._controller.build_wait_timeout,
-            **kwargs)
-
-    def reconfigService(self, name, _, **kwargs):
-        AbstractLatentWorker.reconfigService(
-            self, name, None,
-            build_wait_timeout=self._controller.build_wait_timeout,
-            **kwargs)
+        super(ControllableLatentWorkerMixin, self).__init__(
+            name, None, **kwargs)
```

Comment:
Can you use the python 3 version of `super()` here?

```python
          super().__init__(name, None, **kwargs)
```

That way, it does not matter if this PR or #4620 is merged first.

Reply:
Fixed now.